plugins/modules/monitoring/uptimerobot.py

```diff
@@ -8,138 +8,177 @@
 
 
 DOCUMENTATION = '''
-
 module: uptimerobot
-short_description: Pause and start Uptime Robot monitoring
+short_description: Create, remove, pause and start monitors in Uptime Robot
 description:
-    - This module will let you start and pause Uptime Robot Monitoring
-author: "Nate Kingsley (@nate-kingsley)"
+    - This module lets you create, remove, pause and start monitors in Uptime Robot.
+author:
+- "Nate Kingsley (@nate-kingsley)"
+- "Ninjiner (@Ninjiner)"
 requirements:
     - Valid Uptime Robot API Key
 options:
     state:
         description:
             - Define whether or not the monitor should be running or paused.
+            - C(created) and C(absent) were added in community.general 1.3.0.
+        required: true
+        choices: [ "started", "paused", "created", "absent", "present" ]
+    url:
+        description:
+            - URL to be checked.
         required: true
-        choices: [ "started", "paused" ]
+        version_added: '1.2.0'
```

Comment:
```suggestion
        version_added: '1.3.0'
```
(Same for the others below.)

Reply:
Please add `type: ...` for the new options. That they are not there for the old ones is a sanity error that is ignored for this module, but is something that should also be changed eventually.

Reply:
Changed the version strings: https://github.com/ansible-collections/community.general/pull/961/commits/10ebea24634059a54acd8bdddc4bfba560446209

Reply:
Added data types: https://github.com/ansible-collections/community.general/pull/961/commits/d4d073d4be2a5145d7570f58812eb31a35242bf7